src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -230,6 +230,7 @@ private async Task GenerateDeploymentRecipeSnapShot(Recommendation recommendatio
             recipe.Name = recipeName;
             recipe.CdkProjectTemplateId = null;
             recipe.CdkProjectTemplate = null;
+            recipe.CreatedFromSavedDeploymentProject = true;
 
             var recipeSnapshotBody = JsonConvert.SerializeObject(recipe, Formatting.Indented);
```

Comment:
I noticed that when we serialize the recipe we are writing out every null and empty list and object which looks really ugly. The nulls are an easy fix by passing in a `JsonSerializationSettings` object.

```
var recipeSnapshotBody = JsonConvert.SerializeObject(recipe, new JsonSerializerSettings
{
    Formatting = Formatting.Indented,
    NullValueHandling = NullValueHandling.Ignore,
});
```

To get rid of all of the empty list and objects you will probably need to load the recipe with out the `RecipeDefinition` type because that initializes all objects and lists during the constructor.